.goreleaser.yaml

```diff
@@ -33,9 +33,9 @@ builds:
     ldflags:
       - '-s'
       - '-w'
-      - '-X={{ .ModulePath }}/internal/version.Name=abc'
-      - '-X={{ .ModulePath }}/internal/version.Version={{ .Version }}'
-      - '-X={{ .ModulePath }}/internal/version.Commit={{ .Commit }}'
+      - '-X={{ .ModulePath }}/internal/version.name=abc'
+      - '-X={{ .ModulePath }}/internal/version.version={{ .Version }}'
```

Comment:
Curious to learn why it should be `version.version` instead of `version.Version`, I saw other examples are `version.Version`, see https://github.com/hashicorp/packer-plugin-qemu/blob/1984bb1d2168c671886d40c8a87b341192db5944/.goreleaser.yml#L23. And the official example is goreleaser is `main.version={{.Version}}`, see https://goreleaser.com/customization/builds/#builds